modules/packages/Zarr.chpl

```diff
@@ -50,12 +50,31 @@ module Zarr {
   }
   private use Blosc;
 
+  /* Turns on/off profiling of Zarr IO */
+  config param zarrProfiling = false;
+
+  private var timerDomain: domain(string,parSafe=false) = {"Compression",
```

Comment:
Consider using an enum for this. The main benefit to doing so is that the compiler will tell you if you have misspelled one of these names.

Reply:
I like the strings because they correspond directly with the output; I'll consider this if I add more fine-grain timers later on.